src/tempPolyfills.js

```diff
@@ -0,0 +1,5 @@
+const raf = (global.requestAnimationFrame = cb => {
```

Comment:
What is it?)

Reply:
I get it from youtube lesson. I was trying everything he showed to connect jest and enzyme.

Reply:
Do I have to delete this?

Reply:
Yes, such thing are confusing